### DISCUSSION TITLE
Find or Replace text-based operations in script editor Do NEED auto-capture word under the caret

### DISCUSSION THREAD

**Discussion by jzq740176597** [godotengine/godot-proposals#6632] [Editor]
### Describe the project you are working on

3D

### Describe the problem or limitation you are having in your project

Find or Replace text-based operations in script editor Do NEED auto-capture word under the caret
 **LIKE all main-stream IDE DO**

Or I have to additoinal double-click & Ctrl + C THEN paste in to search or replace! REALLY REALLY tedious!!!

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

I can search and replace symbol very fast & efficient

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

*

### If this enhancement will not be used often, can it be worked around with a few lines of script?

maybe no

### Is there a reason why this should be core and not an add-on in the asset library?

core

**Comment by ghost** [comment]
I've written a plugin that changes this behaviour, so you can just select a word (eg. CTRL + D) and then just press F3/SHIFT + F3 to search for occurrences.  No need to press CTRL + F, etc.  Works beautifully.
